docs/source/daml/intro/9_Functional101.rst

```diff
@@ -115,7 +112,7 @@ If we want to partially apply an infix operation we can also do that as follows:
 
 .. note::
 
-  While function application is left associative by default, infix operators can be declared left or right associative and given a precedence. Good examples are the boolean operations ``&&`` and ``||``, which are declared right associative with precedences 3, and 2, respectively. This allows you to write ``False && True || True && True`` and get value ``True``. See section 4.4.2 of `the Haskell 98 report <https://www.haskell.org/onlinereport/decls.html>`_ for more on fixities. 
+  While function application is left associative by default, infix operators can be declared left or right associative and given a precedence. Good examples are the boolean operations ``&&`` and ``||``, which are declared right associative with precedences 3, and 2, respectively. This allows you to write ``True || True && False`` and get value ``True``. See section 4.4.2 of `the Haskell 98 report <https://www.haskell.org/onlinereport/decls.html>`_ for more on fixities. 
```

Comment:
`False && True || True && True` would still be `True` if we replaced the precedence of `&&` and `||`. I attempted a slightly shorter expression which gives different result with the hypothetical precedences.